worlds/sc2/options.py

```diff
@@ -170,8 +170,17 @@ class MaximumCampaignSize(Range):
 
 class GridTwoStartPositions(Toggle):
     """
-    If turned on and 'grid' or 'hopscotch' mission orders are selected,
-    removes a mission from the starting corner and sets the adjacent two missions as the starter missions.
+    Legacy support for GridTwoStartPositions option.  Use TwoStartPositions on modern settings.
+    """
+    display_name = "Start with two unlocked missions on grid"
+    default = Toggle.option_false
+    visibility = Visibility.none
```

Comment:
Remove it, we don't support YAML compatibility across versions, only for already generated games

Having YAML stability would be a headache to maintain

Reply:
Roger that, removed GridTwoStartPositions and undid most of the changes in regions.py.